src/SDKs/Azure.Core/data-plane/Azure.Core/Azure.Core.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
Is Azure.Core the correct name for this library and package? The description seems to call out the Client Pipeline so wouldn't it be better to include that in the name of the library/package?

Reply:
@KrzysztofCwalina are we going with `Azure.Core` for this package?

Reply:
I will merge with this name and we can rename it before we actually publish the package if we need to.